### PR TITLE
add relationshiptype filter to ajax getcontactrelationships call

### PR DESCRIPTION
**Pull request by mallezie**
Fix for https://issues.civicrm.org/jira/browse/CRM-15982

### CODE REVIEW

**Comment by kurund** [comment]
test this please

File: CRM/Contact/Page/AJAX.php [hunk 1]
```diff
@@ -1073,6 +1074,9 @@ public static function getContactRelationships() {
 
     $params['contact_id'] = $contactID;
     $params['context'] = $context;
+        if($relationship_type_id){
```
Comment (kurund) [inline CRM/Contact/Page/AJAX.php]: code is not indented correctly. Please refer: http://wiki.civicrm.org/confluence/display/CRMDOC/PHP+Code+and+Inline+Documentation

**Comment by mallezie** [comment]
test this please (to see if i can test my own patches)

**Comment by davecivicrm** [comment]
ok to test

**Pull request merged**